src/Architectures.jl

```diff
@@ -86,7 +93,7 @@ function unified_array(::GPU, arr::AbstractArray)
     return vec
 end
 
-## Only for contiguous data!! (i.e. only if the offset for pointer(dst::CuArrat, offset::Int) is 1)
+## Only for contiguous data!! (i.e. only if the offset for pointer(dst::CuArray, offset::Int) is 1)
```

Comment:
What does this comment mean? Is this a warning that the function can be used incorrectly or something? It might be better to enforce more strictly if possible.

Reply:
I have changed the comment